requirements/requirements-articial-dataset.py

```diff
@@ -0,0 +1 @@
+holidays >= 0.9
```

Comment:
Why `>=`? Would be good to have an upper bound

Reply:
I've just copied it from the requirements. I don't think this should be an issue.